Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:4.0
```

Comment:
Adding the supported platform versions will also require a bump in swift-tools-version number from 4.0 to 5.1:

`// swift-tools-version:5.1`

Reply:
I think this was introduced in 5.0 though, so I'm trying 5 instead of 5.1.